src/cljs/org/broadinstitute/firecloud_ui/common/components.cljs

```diff
@@ -80,4 +80,51 @@
                               (when-let [f (:onTabSelected tab)] (f e)))}])
            (:items props))
          [:div {:style {:clear "both"}}]]
-        [:div {} (:component (nth (:items props) (:active-tab @state)))]])}))
+        [:div {} (:component (nth (:items props) (or (:active-tab @state) 0)))]])}))
```

Comment:
Nit: `:active-tab-index` might be clearer. Also, it seems suspicious that you have to check this for `nil` given that it has an initial state set to 0.